topics/engineering/running-background-processes-in-python.gmi

```diff
@@ -0,0 +1,68 @@
+# Running Background Processes in Python using Popen
```

Comment:
Header seems misleading. I think it's just that virtuoso uses the flags `+foreground` to state that the process runs in the foreground. Here's what think these mean:

- foreground: you can actively interact with a process in your terminal. If you press `Ctrl-Z` it's taken to the backgound. Typing `fg` brings it back to the background
- Background: You cannot interact with the process, but it's running/paused in the background. Typing `bg` actually runs the process in your background.
- Daemon: Process ran but detached itself from it's parent and attached itself to P1D 1 (init) hence when we close the terminal, this process won't end.